recipes/ann/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "ann" %}
+{% set version = "1.1.2" %}
+{% set sha256 = "eea03f2e224b66813226d775053316675375dcec45bd263674c052d9324a49a5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url: http://www.cs.umd.edu/~mount/ANN/Files/{{ version }}/{{ name }}_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - patches/cmake.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+test:
+  source_files:
+    - test/test1*
+    - test/test2*
+  commands:
+    - cd test
+    - ann_test < test1.in  # [not win]
+    - ann_test < test2.in  # [not win]
+    - ann_test.exe < test1.in  # [win]
+    - ann_test.exe < test2.in  # [win]
+
+about:
+  home: http://www.cs.umd.edu/~mount/ANN/
+  license: LGPL
```

You should double check but I think [this should be](http://www.cs.umd.edu/~mount/ANN/Files/1.1.2/Copyright_1.1.2.txt):
```suggestion
  license: LGPL-2.1-or-later
  license_family: LGPL
```

---

Yes the Copyright.txt file indicate LGPL 2.1 or later